.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,6 @@
+<!--
+    SPDX-License-Identifier: Apache-2.0
+-->
```

Comment:
I see there's a YAML front matter below. This comment will make it _not_ work. I'd suggest moving the comment into the front matter, using  YAML comments:

```
---
# SPDX-License-Identifier: Apache-2.0

name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''
---

...
```

That's what I did with the Jekyll templates in the `docs` directory.